docs/manifest.json

```diff
@@ -127,6 +127,10 @@
               "title": "Hide Personal Accounts and force organizations",
               "href": "/docs/guides/force-organizations"
             },
+            {
```

Comment:
I think I'd move it/rename it to /docs/organizations/org-slugs-in-urls

Reply:
d5c6100b07826f51ba4d2e5153cdb29ccbb5cfb6

Reply:
for now, let's leave it. i'm trying not to create too many redirects, so that eventually, when the Docs IA occurs, we have less redirects to deal with.